_includes/header.html

```diff
@@ -3,17 +3,22 @@
     <a href="/"><img id="logo" src="/images/fist.svg" alt="1418 logo"></a>
     <div class="header-group">
         <h2 class="title"><a href="/">Vae Victis</a></h2>
-        <h5 class="title-simple"><a href="/">1418</a></h2>
+        <h5 class="title-simple"><a id="title-mini" href="/">TEAM 1418</a></h2>
```

Comment:
This feels a bit clunky. Could you get rid of the `h5` tag and apply all its styling and logic to the `a`?

Reply:
The `h5` tag is needed to get the styling we set for `h5` tags, but I moved the `title-simple` class to the `a` tag